frontend/src/types/AssessmentType.ts

```diff
@@ -0,0 +1,8 @@
+export type AssessmentType = {
+  status: "Draft" | "Published" | "Archived" | "Deleted";
+  name: string;
+  grade: string;
+  type: "Beginning" | "End";
+  country: string;
+  region: string;
+};
```

Comment:
Can we rename this file to `AssessmentTypes` for consistency with the rest of the files in the folder?